libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/navigation/metrics/NavigationFeedbackEvent.kt

```diff
@@ -0,0 +1,28 @@
+package com.mapbox.services.android.navigation.v5.internal.navigation.metrics
+
+import android.annotation.SuppressLint
+import android.location.Location
+import com.mapbox.services.android.navigation.v5.internal.navigation.routeprogress.MetricsRouteProgress
+
+@SuppressLint("ParcelCreator")
+internal class NavigationFeedbackEvent(
+    phoneState: PhoneState,
+    metricsRouteProgress: MetricsRouteProgress
+) : NavigationEvent(phoneState) {
+
+    companion object {
+        private const val NAVIGATION_FEEDBACK = "navigation.feedback"
+    }
+
+    val userId: String = phoneState.userId
```

Comment:
may be here should be smt similar comment https://github.com/mapbox/mapbox-navigation-android/pull/2140/files#r333078017